### ISSUE TITLE
Special Xcode UITesting markup is modified by SwiftFormat

### ISSUE THREAD

**Issue by RabugenTom** [nicklockwood/SwiftFormat#236]
When recording UITests, Xcode generates some special comment markup with alternative choices for tapped elements. For example,

````swift
XCUIApplication()/*@START_MENU_TOKEN@*/.buttons["buttonId"]/*[[".buttons[\"Button\"]",".buttons[\"buttonId\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
````
will be displayed as 
<img width="441" alt="capture d ecran 2018-03-20 a 18 58 19" src="https://user-images.githubusercontent.com/1325273/37685203-55df0d86-2c71-11e8-9724-2d88080502fa.png">

<img width="431" alt="capture d ecran 2018-03-20 a 19 10 18" src="https://user-images.githubusercontent.com/1325273/37685472-614ca5ba-2c72-11e8-871b-b2c2f6e9f22b.png">

SwiftFormat (v0.33.5) adds by default some spaces around the `/*` and `*/` and this prevents Xcode from parsing the markup correctly, and the code is then displayed verbatim:

````swift
XCUIApplication() /* @START_MENU_TOKEN@ */ .buttons["buttonId"] /* [[".buttons[\"Button\"]",".buttons[\"buttonId\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@ */ .tap()
````

**Comment by nicklockwood** [comment]
@RabugenTom hi, sorry it took me a while to get to this. This should be fixed in the next release, however in the meantime you should be able to work around it by putting

```
// swiftformat:disable:next spaceInsideComments spaceAroundComments
```

in the line above `XCUIApplication()`. Alternatively, just put:

```
// swiftformat:disable spaceInsideComments spaceAroundComments
```

at the top of the file, or add `--disable spaceInsideComments spaceAroundComments` to your swiftformat command-line arguments to disable these rules for the whole project.

**Comment by nicklockwood** [comment]
@RabugenTom this is now fixed in version 0.35.3.
